projects/plugins/jetpack/modules/masterbar/inline-help/class-inline-help.php

```diff
@@ -19,20 +19,24 @@ class Inline_Help {
 	 * Inline_Help constructor.
 	 */
 	public function __construct() {
-		$this->register_actions();
+		add_action( 'current_screen', array( $this, 'register_actions' ) );
 	}
 
 	/**
 	 * Registers actions.
 	 *
+	 * @param object $current_screen Current screen object.
 	 * @return void
 	 */
-	public function register_actions() {
+	public function register_actions( $current_screen ) {
 		// phpcs:disable WordPress.Security.NonceVerification.Recommended
 		// Do not inject the FAB icon on embedded screens since the parent window may already contain a FAB icon.
 		$is_framed = ! empty( $_GET['frame-nonce'] );
 
-		if ( $is_framed ) {
+		// Do not inject the FAB icon on Yoast screens to avoid overlap with the Yoast help icon.
+		$is_yoast = false !== strpos( $current_screen->base, 'wpseo' );
```

Comment:
Is it worth adding a check to ensure `$current_screen->base` is set before we use it, to avoid issues with other plugins that may play with `$current_screen`?

Reply:
Yeah that's a good idea, added here: https://github.com/Automattic/jetpack/pull/19890/commits/218853e74cee2b4a68ee1fbeb9add422933d1551